sdk/openai/Azure.AI.OpenAI/src/Custom/AzureOpenAIClientBuilderExtensions.cs

```diff
@@ -1,43 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The custom versions were no longer necessary with the signature unification/changes (making method-level deployment/model specification required again); the generated one handles all the remaining cases.

We'll need the custom counterparts again if/when we re-add in initialization-time deployment/model specification (for defaults or otherwise), but for the time being it gets to be the one bright spot of custom code *reduction* in this PR!